pivy-importer/src/main/groovy/com/linkedin/python/importer/ivy/IvyFileWriter.groovy

```diff
@@ -86,15 +88,15 @@ class IvyFileWriter {
             def map = [name: name, ext: ext, conf: source ? 'source' : 'default', type: ext]
 
             if (filename.indexOf(version) + version.length() + 1 < filename.length()) {
-                map['m:classifier'] = getClassifier(filename)
+                map['m:classifier'] = getClassifierFromFilename(filename)
             }
             return map
         }
 
         return publicationMap
     }
 
-    private String getClassifier(String filename) {
+    private String getClassifierFromFilename(String filename) {
```

Comment:
I personally think getClassifier() was a more accurate name.  It doesn't technically have to be from a filename - the string parsing being done in here works equally well on any suitably-formatted string.  This also falls into the gratuitous change bucket.

Reply:
Sounds good to me.